source/_docs/mqtt/broker.markdown

```diff
@@ -31,6 +31,13 @@ Home Assistant contains an embedded MQTT broker. If no broker configuration is g
 mqtt:
 ```
 
+<p class='note warning'>
+There is an issue with the HBMQTT broker that can cause a memory leak (slowly increasing used memory). This causes an unstable system after the memory is full. You could measure/monitor this with a system monitor. The issue is from 2016 and could already be resolved with newer versions. Use another broker when you experience this issue, for example, Mosquitto. <br>
+<br>
+Issue with the HBMQTT broker: https://github.com/beerfactory/hbmqtt/issues/62  <br>
```

Comment:
We should use formatting for links

![image](https://user-images.githubusercontent.com/1444314/44021344-d4f700de-9ee4-11e8-9134-96d1e7a2e748.png)

Reply:
Link formatting was not working at the time, see you already fixed it and made it smaller. 👍